src/oci/config.py

```diff
@@ -111,7 +111,7 @@ def validate_config(config, **kwargs):
     errors = {}
     for required_key in REQUIRED:
         fallback_key = REQUIRED_FALLBACKS.get(required_key)
-        if required_key not in config and fallback_key not in config:
+        if (required_key not in config or config[required_key] is None) and fallback_key not in config:
```

Comment:
if we're doing this to handle `config[required_key] = None` I think it makes sense to make the same fix for  `config[fallback_key] = None`

Reply:
True. Thanks for pointing it out @mross22. 
I've updated the branch with changes.